packages/bundler/src/MethodHandlerRIP7560.ts

```diff
@@ -0,0 +1,46 @@
+import { JsonRpcProvider, TransactionReceipt } from '@ethersproject/providers'
+import {
+  AddressZero,
+  getRIP7560TransactionHash,
+  OperationRIP7560,
+  requireCond,
+  tostr
+} from '@account-abstraction/utils'
+import { ExecutionManager } from './modules/ExecutionManager'
+import { HEX_REGEX } from './MethodHandlerERC4337'
+
+export interface RIP7560TransactionReceipt extends TransactionReceipt {
+
+}
+
+export class MethodHandlerRIP7560 {
+  constructor (
+    readonly execManager: ExecutionManager,
+    readonly provider: JsonRpcProvider
+  ) {}
+
+  async sendRIP7560Transaction (transaction: OperationRIP7560): Promise<string> {
+    await this._validateParameters(transaction)
+    console.log(`RIP7560Transaction: Sender=${transaction.sender}  Nonce=${tostr(transaction.nonce)} Paymaster=${transaction.paymaster ?? ''}`)
+    await this.execManager.sendUserOperation(transaction, '')
+    return getRIP7560TransactionHash(transaction)
+  }
+
+  async getRIP7560TransactionReceipt (txHash: string): Promise<RIP7560TransactionReceipt | null> {
+    requireCond(txHash?.toString()?.match(HEX_REGEX) != null, 'Missing/invalid userOpHash', -32601)
+    return await this.provider.getTransactionReceipt(txHash)
+  }
+
+  // TODO: align parameter names across 4337 and 7560
+  async _validateParameters (transaction: OperationRIP7560): Promise<void> {
+    (transaction as any).deployer = transaction.factory ?? AddressZero;
```

Comment:
why force ZEROADDR for missing params ?

Reply:
I think there was some weird null pointer error downstream, I can try removing and debugging but I don't want to. Is this a problem?

Reply:
yes, I think we should fix downstream errors, and not rely on zero values.

Reply:
Will create a separate issue for this, as well as other "shape" validation checks. I don't think this issue should be a blocker for the PR.